File: Assignment_41/Assignment_41_3.c
#include<stdio.h>

int Strlen(char str[])
{
    int iCount = 0;
    //O(n) 
    while(*str != '\0')
    {
        if((*str >= 'A' )&& (*str <= 'Z') || (*str >= 'a') && (*str <= 'z'))
        {
             iCount++;
        }
        str++;
    }
    
    return iCount;
}
// Main Function
int main()
{
    char Arr[50] = {'\0'};
    int iRet = 0;

    printf("Enter String :\n");
    scanf("%[^'\n']s",Arr);
                            
   iRet = Strlen(Arr);

   printf("Number of Character Present in string : %d\n",iRet);

    return 0;
}

File: Assignment_41/Assignment_41_3.java
import java.util.*;

class StringX
{
    public int strlenX(String str)
    {
        int iCnt = 0, iCount = 0;

        char Arr[] = str.toCharArray();
        //O(n) 
        for(iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            iCount++;
        }

        return iCount;
    }
}

class Assignment_41_3
{
    public static void main(String A[])
    {
        int iRet = 0;

        Scanner scanobj = new Scanner(System.in);

        System.out.println("Enter string : ");
        String sobj = scanobj.nextLine();

        StringX strobj = new StringX();
        
        iRet = strobj.strlenX(sobj);

        System.out.println("Number of caharacter Present in String :"+iRet);
    }
}